application/views/template_view.py

```diff
@@ -4,17 +4,19 @@
 template_view = Blueprint('template_view', __name__)
 
 
-@template_view.route('/template/<template_id>', methods=['POST'])
+@template_view.route('/template/<template_id>', methods=['POST', 'PUT'])
```

Comment:
Would the code be more explicit by splitting these out into different routes. The code dealing with creating the data should really have to know about the http method used. It might be cleaner to have something like

```
@template_view.route('/template/<template_id>', methods='POST', 'PUT')
def create_template(template_id):
    template_controller.create_comms_template(template_id, template=request.get_json())
    return Response(status=201)

@template_view.route('/template/<template_id>', methods='PUT')
def update_template(template_id):
    template_controller.update_comms_template(template_id, template=request.get_json())
    return Response(status=200)
```

Simple is better than complex. - Less boolean logic = simplicity
Flat is better than nested. - Nesting logic within boolean checks
Readability counts. - The less routes to follow the easier it is to read